clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -311,6 +321,14 @@ func createNamespacesList(ctx Context, namespaceRange *api.NamespaceRange) []str
 	return nsList
 }
 
+func updateResourceVersion(ctx Context, obj *unstructured.Unstructured) error {
+	identifier := state.ResourceTypeIdentifier{
+		ObjectKind: obj.GetKind(),
+		ApiGroup:   obj.GetAPIVersion(),
```

Comment:
GetAPIVersion is not ApiGroup.

GetAPIVersion returns "group/version", which is not exactly what we want.